### DISCUSSION TITLE
Add Linux support to df-steam-hook?

### DISCUSSION THREAD

**Discussion by malacration** [dfint/.github#14] [Q&A] [answered]
Could this project have a Linux build?

**Comment by insolor** [comment, marked as answer]
See https://github.com/dfint/df-steam-hook-rs, it has both windows and linux support. But it is in work-in-progress state at the time.

Update: https://github.com/dfint/df-steam-hook-rs is ready to use, and https://github.com/dfint/installer installs it along with all needed files.

**Comment by malacration** [comment]
Is it not possible to build c++ on Linux?

**Comment by insolor** [comment]
No, it depends on [detours](https://github.com/microsoft/Detours) library, which is windows-only.

**Comment by malacration** [comment]
thanks for answer!
